mesh_sequence_controller.py

```diff
@@ -166,6 +185,7 @@ def loadSequenceFromFile(self, _obj, _dir, _file):
         numFrames = 0
         unsortedFiles = glob.glob(full_filepath)
         sortedFiles = sorted(unsortedFiles)

```

Comment:
We probably don't need to call `sorted()` anymore.

Reply:
`sort_nicely()` removed and key parameter is passed to sorted.